source/materials/XenonLiquidProperties.cc

```diff
@@ -127,7 +127,7 @@ namespace nexus {
   void XenonLiquidProperties::Scintillation
   (std::vector<G4double>& energy, std::vector<G4double>& intensity)
   {
-    for (G4int i=0; i<energy.size(); i++)
+    for (unsigned int i=0; i<energy.size(); i++)
```

Comment:
I think you should delete `int` here

Reply:
Only for aesthetic purposes; it makes no difference to the semantics: https://en.cppreference.com/w/cpp/language/types

Reply:
Nice! I was not sure!

Reply:
Actually, I *should* have told you to use `size_t`, which is the type specifically designed for this purpose. Sorry, I wasn't fully concentrating.